cl/api/views.py

```diff
@@ -213,6 +213,11 @@ async def deprecated_api(request, v):
     )
 
 
+def rest_change_log(request):
+    context = {"private": False}
+    return render(request, "rest-change-log.html", context)
```

Comment:
There are tests in `BasicAPIPageTest` that just load these very basic URLs. Mind adding this URL too?